GHzDACs/ghz_fpga_server.py

```diff
@@ -607,13 +612,15 @@ def run(self, runners, reps, setupPkts, setupState, sync, getTimingData,
         # setupState is a set
         setupPkts.extend(pkt for pkt, state in boardSetupPkts)
         setupState.update(state for pkt, state in boardSetupPkts)
-        
+
         try:
             yield self.pipeSemaphore.acquire()
+            logging.info('pipe semaphore acquired')
```

Comment:
Re: the logging levels. I would like to put them at the debug level, but I used the debug level for dumping packets, so I used the info level for "how far in run_sequence did we get this time". Then we put the default level to warning (which I've moved to the server start up), and I think it's reasonable.